Src/Services/ActionService.lua

```diff
@@ -53,7 +57,52 @@ end
 
 function ActionService.OnStart(self: ActionService)
 	MessageReceiveService.OnAction:Connect(function(packet: ActionPacket)
-		-- fixme: account for 'behaviour' key in packet
+		if packet.behaviour == ActionBehaviour.SelectedServers or packet.behaviour == ActionBehaviour.SelectedServer then
```

Comment:
If it is selected server then it has to claim. That's the difference between selected servers and selected server (singular) (i just remembered)

Reply:
and place id vs place ids

Reply:
oh, ok